CHANGELOG.md

```diff
@@ -2,6 +2,35 @@
 
 Friendly reminder: We have a [bug bounty program](https://hackerone.com/cosmos).
 
+## v0.35.8
+
+July 12, 2022
+
+Special thanks to external contributors on this release: @joeabbey
```

Comment:
Even if we don't elaborate further on specific changelog entries, can we please look at, from now on, adding a release summary for each release? It should focus on the main value-adds users would potentially be interested in. Something like:

```suggestion
This release primarily fixes a bug in the priority mempool that resulted in unbounded heap growth when under load.

As a temporary measure toward reducing disk space for a pruning node, this release also introduces a utility command to perform a compaction on GoLevelDB-based state and block stores. This utility command will be removed in future after the planned storage engine refactor.

Special thanks to external contributors on this release: @joeabbey
```

Reply:
That seems like a reasonable idea. Maybe consider making a PR to amend the RELEASES doc to include that.